### PR TITLE
I1715 - don't override credentials by default

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: setup_starport.sh [hunk 1]
```diff
@@ -31,6 +31,21 @@ function cleanup {
 
 trap cleanup EXIT
 
+while getopts f o; do
```
Comment (richfitz) [inline setup_starport.sh:39]: How does this work? Is this triggered by a `-f` flag? Can that be added as a usage statement somewhere?

**Review by richfitz — CHANGES_REQUESTED** [review]

File: setup_starport.sh (hunk 1 above)
Comment (hillalex) [reply setup_starport.sh:39]: it's sorta self documenting here
https://github.com/vimc/bb8/pull/26/files#diff-dbf6b7c6b0c7104129d222fc3f07147bR41

do you think that's enough or should I add it somewhere else too?

**Review by MartinEden — APPROVED** [review]
Looks good, other than the thing Rich picked up. This does feel like the whole script could do with a Python rewrite, but that's out of scope for now, I think.

File: setup_starport.sh (hunk 1 above)
Comment (richfitz) [reply setup_starport.sh:39]: If you could add a general "Usage:" section perhaps, but otherwise that makes sense to me

**Comment by hillalex** [comment]
Yeah I considered re-writing in python. We're probably just at the limit of what we want to do in bash here!

File: setup_starport.sh (hunk 1 above)
Comment (hillalex) [reply setup_starport.sh:39]: Cool, added at the top of this file

**Review by richfitz — APPROVED** [review]
Agreed that this should eventually go into python.  Perhaps we should rename to `setup_starport` so that we can do that later without changing anything else?  Otherwise LGTM

**Pull request merged**